File: robot/Comm/Messages/Remote/ToRobot/StartRemoteControlMessage.cpp
#include "StartRemoteControlMessage.h"

#include <cstring>

StartRemoteControlMessage::StartRemoteControlMessage()
  : RemoteMessageToRobot(RemoteMessageToRobot::startRemoteControl, 0, NULL) {
}

StartRemoteControlMessage::~StartRemoteControlMessage() {
}

File: tools/SharedJavaLibrary/src/network/message/torobot/StartRemoteControlMessage.java
package network.message.torobot;

import network.message.RemoteMessageToRobot;

public class StartRemoteControlMessage extends RemoteMessageToRobot {

  public StartRemoteControlMessage() {
    super(RemoteMessageToRobot.MessageType.START_REMOTE_CONTROL, null);
  }
  
}
